test/integration/bin/multi-path/index.test.ts

```diff
@@ -21,6 +20,8 @@ describe('integration', () => {
             '#!/usr/bin/env node',
           )
         }
+
+        await deleteFile(`${__dirname}/fixtures/tsconfig.json`)
```

Comment:
Kinda forgot the reason why we need to remove them here?

I though if we have 2 integration tests share the same directory, any required to delete files can be remove in the beforeEach/afterEach jest lifecycle.

But not sure if we still need them here for single test?

Reply:
My bad, for a single test, absolutely not.